lstchain/scripts/lstchain_dl1_to_dl2.py

```diff
@@ -45,47 +46,46 @@
 from lstchain.reco import dl1_to_dl2
 from lstchain.reco.utils import filter_events, impute_pointing, add_delta_t_key
 
-parser = argparse.ArgumentParser(description="DL1 to DL2")
+parser = argparse.ArgumentParser(description="""Run the DL1 to DL2 step, applying prediction from the trained models.
```

Comment:
This will look very strange in the description, since the multiline string will contain the indentation.

Better put this description at the top of the module (what will become `__doc__`) and do `description=__doc__`.